code/game/gamemodes/wizard/soulstone.dm

```diff
@@ -58,6 +68,10 @@
 	. = ..()
 	if(iscultist(user) && purified && !iswizard(user))
 		to_chat(user, "<span class='danger'>[src] reeks of holy magic. You will need to cleanse it with a ritual dagger before anything can be done with it.</span>")
+		return
+	if(user.mind?.isholy)
+		to_chat(user, "<span class='danger'>An overwhelming feeling of dread comes over you as you pick up [src]. It looks fragile enough to break with your hands.</span>")
+		return
```

Comment:
This message will play to chaplains when they pick up their mysterious old shard. Is that intended?

Reply:
Yes, just a hint that they can break the shard :)